src/Microsoft.ML.Transforms/Text/TextStaticExtensions.cs

```diff
@@ -8,6 +8,8 @@
 using Microsoft.ML.Runtime.Data;
 using System;
 using System.Collections.Generic;
+using static Microsoft.ML.Runtime.TextAnalytics.StopWordsRemoverTransform;
```

Comment:
Ctrl+R+G

Reply:
This is sorted already.

---
In reply to: [219550492](https://github.com/dotnet/machinelearning/pull/953#discussion_r219550492) [](ancestors = 219550492)